src/Display/Display.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+// This is a div element that is in font-size bigger than h1 and is supposed to
+// used as 'slogan' around our apps. We dont want this to be crawled from search engines
+// since its only for design reasons but instead we want the h1 to be so as it is meant by default.
```

Comment:
```suggestion
// since its only for design reasons.
```
I think we can drop the end of this (or rewrite it?)